terra-jupyter-python/Dockerfile

```diff
@@ -20,69 +20,82 @@ RUN apt-get update && apt-get install -yq --no-install-recommends \
 
 ENV HTSLIB_CONFIGURE_OPTIONS="--enable-gcs"
 
+# Dev note: in general, do not pin Python packages to any particular version.
+# Depend on the smoke tests to help us identify any package incompatibilties.
+#
+# If we find that we do need to pin a package version, be sure to:
+# 1) Add a comment saying what needs to be true for us to remove the pin.
+#    (e.g. link to an issue and put the details there)
+# 2) If the smoke tests did not show the problem, add a new test case to improve
+#    test coverage for the identified problem.
 RUN pip3 -V \
  && pip3 install --upgrade pip \
- && pip3 install numpy==1.15.2 \
- && pip3 install py4j==0.10.7 \
- && python3 -mpip install matplotlib==3.0.0 \
- && pip3 install pandas==0.25.3 \
- && pip3 install pandas-gbq==0.12.0 \
- && pip3 install pandas-profiling==2.4.0 \
- && pip3 install seaborn==0.9.0 \
- && pip3 install python-lzo==1.12 \
- && pip3 install google-cloud-bigquery==1.23.1 \
- && pip3 install google-api-core==1.6.0 \
- && pip3 install google-cloud-bigquery-datatransfer==0.4.1 \
- && pip3 install google-cloud-datastore==1.10.0 \
- && pip3 install google-cloud-resource-manager==0.30.0 \
- && pip3 install google-cloud-storage==1.23.0 \
- && pip3 install scikit-learn==0.20.0 \
- && pip3 install statsmodels==0.9.0 \
- && pip3 install ggplot==0.11.5 \
- && sed -i 's/pandas.lib/pandas/g' /usr/local/lib/python3.7/dist-packages/ggplot/stats/smoothers.py \
- # the next few `sed` lines are workaround for a ggplot bug. See https://github.com/yhat/ggpy/issues/662
```

Comment:
Apparently this is fixed (I don't even remember this) -- I see there is a `pandas` test case 👍